client/state/data-layer/wpcom/me/send-verification-email/index.js

```diff
@@ -11,22 +11,30 @@ import {
 } from 'state/action-types';
 
 import { registerHandlers } from 'state/data-layer/handler-registry';
+import { translate } from 'i18n-calypso';
+import { infoNotice, errorNotice, successNotice, removeNotice } from 'state/notices/actions';
+
+const infoNoticeId = 'email-verification-sending';
 
 /**
  * Creates an action for request for email verification
  *
  * @param 	{object} action The action to dispatch next
  * @returns {object} Redux action
  */
-export const requestEmailVerification = ( action ) =>
+export const requestEmailVerification = ( action ) => [
+	...( action.showGlobalNotices
+		? [ infoNotice( translate( 'Sending email…' ), { id: infoNoticeId, duration: 4000 } ) ]
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Sending email' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/home-janitorial-enhancements)